src/udiskslinuxdrive.c

```diff
@@ -40,6 +40,7 @@
 #include <mntent.h>
 
 #include <glib/gstdio.h>
+#include <libstoragemgmt/libstoragemgmt.h>
```

Comment:
Since you guys are more focusing on laptop and workstation, it make me think whether I should bury this PR or not.

Let me find some real user on this before we spend more time on this.